src/main/menu.js

```diff
@@ -13,6 +13,9 @@ import { checkForUpdates } from './lib/updater';
 import { getMainWindow } from './utils/window';
 import i18n, { languages } from '../shared/i18n';
 import pkg from '../../package.json';
+import electronContextMenu from 'electron-context-menu';
+
+electronContextMenu();
```

Comment:
@sallar Is this still needed?

Reply:
@perry-mitchell yeah, this was added by @hugo-cardenas and we discussed it in the initial PR. it adds some useful stuff -> https://github.com/buttercup/buttercup-desktop/pull/454#discussion_r155945312